cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/DefaultSchedulerService.java

```diff
@@ -66,8 +66,8 @@ public void execute(JobExecutionContext context) throws JobExecutionException {
       Trigger trigger = context.getTrigger();
       String key = trigger.getKey().getName();
       String[] parts = key.split(":");
-      Preconditions.checkArgument(parts.length == 6, "Trigger's key name {} has {} parts instead of 6",
-                                  key, parts.length);
+      Preconditions.checkArgument(parts.length == 6, String.format("Trigger's key name %s has %d parts instead of 6",
```

Comment:
Yes, when we read the triggers from the store, we add the default version to it. And so yes, checkArgument is still valid.